extensions/dirvish-emerge.el

```diff
@@ -5,6 +5,7 @@
 ;; Version: 1.8.14
 ;; Keywords: files, convenience
 ;; Homepage: https://github.com/alexluigit/dirvish
+;; Package-Requires: ((emacs "28.1"))
 ;; SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
A Package-Requires declaration is only needed in the main entry (dirvish.el), package-lint would raise an error if this line exists since this file is part of the package recipe.

Reply:
Sorry, this wasn't meant to end up in the pr. It was just so I could get sensible warning from `flycheck`. But I will wait till till pr is ready before removing it if that is ok.